emails/templates/emails/first_time_user.html

```diff
@@ -2,8 +2,10 @@
   Note that Django only loads strings from some Fluent files.
   See privaterelay/ftl_bundles.py.
 {% endcomment %}
+{% load cache %}
 {% load ftl %}
 {% load email_extras %}
+{% cache 86400 welcome_email_html language %}
```

Comment:
(suggestion, non-blocking) Consider setting a cache timeout variable in the context to be used in the templates so we can use a single variable and expect same cache behavior for caching email templates.

Reply:
I created https://mozilla-hub.atlassian.net/browse/MPP-3220 for a follow-up.